includes/i18n.php

```diff
@@ -175,11 +201,16 @@
 		#endregion Plugin Deactivation
 
 		#region Connect
-		'hey-x'                                    => _x( 'Hey %s,', 'greeting', 'freemius' ),
-		'thanks-x'                                 => _x( 'Thanks %s!', 'a greeting. E.g. Thanks John!', 'freemius' ),
-		'connect-message'                          => __( 'In order to enjoy all our features and functionality, %s needs to connect your user, %s at %s, to %s', 'freemius' ),
-		'connect-message_on-update'                => __( 'Please help us improve %2$s! If you opt-in, some data about your usage of %2$s will be sent to %5$s. If you skip this, that\'s okay! %2$s will still work just fine.', 'freemius' ),
-		'pending-activation-message'               => __( 'You should receive an activation email for %s to your mailbox at %s. Please make sure you click the activation button in that email to complete the install.', 'freemius' ),
+		'hey-x'                                    => /* translators: %s: User name (e.g. John) */
+		                                              __( 'Hey %s,', 'freemius' ),
+		'thanks-x'                                 => /* translators: %s: User name (e.g. John) */
+		                                              __( 'Thanks %s!', 'freemius' ),
+		'connect-message'                          => /* translators: 1: Plugin name 2: ??? 3: ??? 4: ??? */
```

Comment:
@ramiy:

2: Admin username
3: Current site's domain
4: Link to `freemius.com`